Cartfile

```diff
@@ -0,0 +1,2 @@
+github "ReactiveX/RxSwift" ~> 4.0
```

Comment:
Does this also pull RxCocoa? I think RxSwift works with Linux which doesn't have any Cocoa frameworks